### DISCUSSION TITLE
folder/file list options; move commands

### DISCUSSION THREAD

**Discussion by EricEWeir** [mikekazakov/nimble-commander#149] [Q&A] [answered]
currently my folder/file lists display only names. none of the other options, e.g., date created, etc. I didn't notice this before. maybe I had them previously and somewhere/somehow the settings got changed. 

also, on the list of command key maps I find maps for several "move" commands: "move item here," "move to," and "move as." "move item here" is mapped to opt-cmd-v; i don't know what to expect; I see no effect. "move to" is mapped to F6; F6 doesn't do anything on my installation. "move as" is mapped to shift-F6; I also see no effect from it.

**Comment by mikekazakov** [comment, marked as answer]
> currently my folder/file lists display only names. none of the other options, e.g., date created, etc. I didn't notice this before. maybe I had them previously and somewhere/somehow the settings got changed.

There are different layouts for list view in NC, that's customisable.
You can switch them via e.g. Ctrl+3/Ctrl+4 or adjust the shown columns via right-click on the header:
![image](https://github.com/mikekazakov/nimble-commander/assets/7439287/18952b37-3f7a-4500-84c5-da9e7e0a6f3b)

> also, on the list of command key maps I find maps for several "move" commands: "move item here," "move to," and "move as." "move item here" is mapped to opt-cmd-v; i don't know what to expect; I see no effect. "move to" is mapped to F6; F6 doesn't do anything on my installation. "move as" is mapped to shift-F6; I also see no effect from it.

Opt+Cmd+V moves an item selected in the clipboard into the current directory, opposing Cmd+V which would copy that item there.

F6 moves the items selected in the focused panel to the opposite panel:
![image](https://github.com/mikekazakov/nimble-commander/assets/7439287/9f9a367a-9443-4750-ae98-186441c4d5ce)

Shift+F6 allows to rename in the same directory (somewhat similar to what Ctrl+F6 does, but with a full-blown dialog):
![image](https://github.com/mikekazakov/nimble-commander/assets/7439287/f031d875-0ddd-4847-b0c5-14b18de7b76e)

Hope that helps a bit.

**Reply by EricEWeir** [reply]
thanks, mike. yes, those address all my questions. i see how"move" the commands work now.
